zds/tutorialv2/views/validations_opinions.py

```diff
@@ -126,26 +127,28 @@ def form_valid(self, form):
             )
 
             bot = get_object_or_404(User, username=settings.ZDS_APP["member"]["bot_account"])
-            if not self.object.validation_private_message:
-                self.object.validation_private_message = send_mp(
-                    bot,
-                    versioned.authors.all(),
-                    self.object.validation_message_title,
-                    versioned.title,
-                    msg,
-                    send_by_mail=True,
-                    direct=False,
-                    hat=get_hat_from_settings("moderation"),
-                )
-                self.object.save()
-            else:
+            if self.object.validation_private_message:
                 send_message_mp(
                     bot,
                     self.object.validation_private_message,
                     msg,
                     hat=get_hat_from_settings("moderation"),
                     no_notification_for=[self.request.user],
                 )
+            else:
+                recipients = filter_reachable(versioned.authors.all())
+                if len(recipients) > 0:
+                    self.object.validation_private_message = send_mp(
+                        bot,
+                        versioned.authors.all(),
```

Comment:
Ça ne devrait pas être `recipients` ici ?
```suggestion
                        recipients,
```

Reply:
Bien vu, corrigé !